libs/image/include/image/ColorTransform.h

```diff
@@ -155,8 +155,8 @@ std::unique_ptr<uint8_t[]> fromLinearTosRGB(const LinearImage& image) {
         float const* p = image.getPixelRef(0, y);
         for (size_t x = 0; x < w; ++x, p += nchan, d += N) {
             for (int n = 0; n < N; n++) {
-                float source = linearTosRGB(p[n]);
-                float target = math::saturate(source) * std::numeric_limits<T>::max();
+                float source = n < 3 ? linearTosRGB(p[n]) : p[n];
```

Comment:
We should also check the disassembly for this. I imagine clang will just unroll the loop, otherwise we could do this manually.